com.qq.QQmusic.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.qq.QQmusic</id>
+  <provides><id>com.qq.QQmusic.desktop</id></provides>
+  <launchable type="desktop-id">com.qq.QQmusic.desktop</launchable>
+  <name>QQmusic</name>
+  <name xml:lang="zh-cn">QQ音乐</name>
+  <developer_name>tencent</developer_name>
+  <summary>Online music streaming service</summary>
+  <summary xml:lang="zh-cn">QQ音乐</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=http://y.qq.com/y/static/tips/service_tips.html</project_license>
+  <url type="homepage">https://y.qq.com/</url>
+  <url type="help">https://y.qq.com/download/welcome_pc_v15/index.html?ADTAG=YQQ</url>
+  <description>
+    <p>Enjoy your online music.</p>
```

Comment:
I'd change it to something like this: "QQMusic is an online music streaming service operating in China".